experimental/traffic-portal/src/app/api/profile.service.ts

```diff
@@ -71,6 +71,22 @@ export class ProfileService extends APIService {
 		return this.get<Array<ResponseProfile>>(path).toPromise();
 	}
 
+	/**
+	 * Retrieves Profiles associated with a Parameter from the API.
+	 *
+	 * @param id Specify the integral, unique identifier (number) of a specific Parameter, for which the Profiles are to be retrieved.
+	 * @returns The requested Profile(s).
+	 */
+	public async getProfilesByParam(id: number): Promise<Array<ResponseProfile>> {
+		const path = "profiles";
+		if (id !== undefined) {
+			const params = {param: String(id)};
+			const r = await this.get<[ResponseProfile]>(path, undefined, params).toPromise();
```

Comment:
This templated type says that this will always be a length = 1 array, but I don't think that's true, in general. Could be empty, or could have a thousand entries.